src/mqtt/qmqtt_client_p.cpp

```diff
@@ -292,11 +303,15 @@ void QMQTT::ClientPrivate::startKeepAlive()
 {
     _timer.setInterval(_keepAlive*1000);
     _timer.start();
+    // The MQTT specification does not mention a timeout value in this case, so we use 10% of the
```

Comment:
Is it possible to set a ceiling here? For example: 10 seconds.

Reply:
I don't mind.